src/plugins/opensearch_dashboards_overview/public/components/overview/overview.tsx

```diff
@@ -149,10 +149,12 @@ export const Overview: FC<Props> = ({ newsFetchResult, solutions, features, bran
         hideToolbar={isNewOpenSearchDashboardsInstance}
         iconType="inputOutput"
         title={
-          <FormattedMessage
-            defaultMessage="OpenSearch Dashboards"
-            id="opensearchDashboardsOverview.header.title"
-          />
+          branding.applicationTitle || (
+            <FormattedMessage
+              defaultMessage="OpenSearch Dashboards"
+              id="opensearchDashboardsOverview.header.title"
+            />
+          )
         }
```

Comment:
I believe we can just utilize the `<FormattedMessage>` like below. The default of branding.applicationTitle is "OpenSearch Dashboards".

```
title={
            <FormattedMessage
              defaultMessage={branding.applicationTitle}
              id="opensearchDashboardsOverview.header.title"
            />
     }
```